src/oci_features.c

```diff
@@ -217,7 +219,9 @@ crun_features_add_annotations_info (yajl_gen json_gen, const struct annotations_
   yajl_gen_string (json_gen, (const unsigned char *) "annotations", strlen ("annotations"));
   yajl_gen_map_open (json_gen);
 
+#ifdef HAVE_SECCOMP
   add_string_to_json (json_gen, "io.github.seccomp.libseccomp.version", annotation->io_github_seccomp_libseccomp_version);
```

Comment:
should this be done only when `if (! is_empty_string (annotation->io_github_seccomp_libseccomp_version))` ?